src/message/customer-information/index.tsx

```diff
@@ -18,6 +21,11 @@ export const CustomerInformation: React.FC<GetCustomerProps> = ({
   };
 
   const getCustomersRequest = new CustomerInformationRequest();
+  const idToken = new IdToken();
+  idToken[IdTokenProps.id] = NEW_IDENTIFIER as any;
+  getCustomersRequest[GetCustomerInformationDataProps.clear] = false;
```

Comment:
I dont think you need to set these two because booleans default to false automatically

Reply:
@elliot-sabitov it was defaulting to undefined for me which is why I set them as a default